content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Notation v1.0.0-RC.4 is available!
+author:  "Notation Release Team"
+date: 2023-04-21
+draft: false
+---
+
+The Notation maintainers are pleased to announce the releases of Notation CLI v1.0.0-rc.4, notation-go v1.0.0-rc.4, and notation-core-go v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release adds the following significant enhancements:
+
+- Support validating certificate revocation with [Online Certificate Status Protocol](https://datatracker.ietf.org/doc/html/rfc6960) (OCSP)
+- Introduce switch `NOTATION_EXPERIMENTAL=1` to enable experimental features
+- Introduce new CLI command `notation policy` to simplify trust policy configuration
+- Support OCI distribution referrers API
+- Introduce signing, listing and verification with [OCI image layout](https://github.com/opencontainers/image-spec/blob/v1.0/image-layout.md) as experimental feature
```

Comment:
I think, we should expand on this definition to say  -  Introduce signing, listing and verification with OCI image layout **for signing images before they are pushed in a registry** as an experimental feature